credentials_manager/lib/credentials_manager/account_manager.rb

```diff
@@ -13,7 +13,8 @@ def initialize(user: nil, password: nil, prefix: nil)
     end
 
     def user
-      @user ||= fetch_password_from_env
+      @user ||= ENV["FASTLANE_USER"]
```

Comment:
this is the user method. so we want to get the user from env vars, not the password. right?